### DISCUSSION TITLE
About the SPREAD_RATE setting

### DISCUSSION THREAD

**Discussion by abc-k** [firemodels/fds#12255] [Q&A] [answered]
Dear,
I want to simulate a spreading flame. So I added a SPREAD_RATE with RAMP_Q input to the SURF line.
However, when I run the simulation in this state, the behavior of the HRR no longer matches the RAMP_Q input.
![image](https://github.com/firemodels/fds/assets/150226734/adba2502-19fb-424d-b722-334b9ceb4dd3)

When SPREAD_RATE is not specified, it is confirmed to follow the input of RAMP_Q.

When using SPREAD_RATE, why is the output of HRR different from the input of RAMP_Q?

**Comment by abc-k** [comment]
&HEAD CHID='SPREAD', TITLE=''  /

&MESH IJK=40,40,20 XB=0.0,10,0.0,10,0.0,5/
&TIME T_END=60. /

&DUMP MASS_FILE=T /

&REAC ID = 'N-HEPTANE'
	  FUEL='N-HEPTANE'
	  CO_YIELD = 0.1
	  SOOT_YIELD = 0.1/
	 

!SURF 
&SURF ID='N-HEPTANE',
      COLOR='RED',
      SPEC_ID='N-HEPTANE'
      HRRPUA=1000
      RAMP_Q='fire'
      XYZ=5,5,2,
  **SPREAD_RATE=0.25/**
&RAMP ID='fire'T=0,F=0/
&RAMP ID='fire'T=10,F=0/
&RAMP ID='fire'T=30,F=1/
&RAMP ID='fire'T=40,F=1/
&RAMP ID='fire'T=50,F=0/
&RAMP ID='fire'T=60,F=0/
     

!BURN OBST
&OBST ID='BURNER' ,XB= 2,8,4,6,0,2,SURF_ID6(6)='N-HEPTANE'/
&VENT XB=0,10,0,10,5,5,SURF_ID='OPEN'/

&DEVC ID='FUEL_CENTER', XYZ=5,5,2,  QUANTITY='MASS FRACTION',SPEC_ID='N-HEPTANE'/
&DEVC ID='FUEL_R'     , XYZ=5,8,2,  QUANTITY='MASS FRACTION',SPEC_ID='N-HEPTANE'/
&DEVC ID='FUEL_L'     , XYZ=5,2,2,  QUANTITY='MASS FRACTION',SPEC_ID='N-HEPTANE'/
&DEVC ID='FUEL_R_1'   , XYZ=5,6.5,2,QUANTITY='MASS FRACTION',SPEC_ID='N-HEPTANE'/
&DEVC ID='FUEL_L_1'   , XYZ=5,3.5,2,QUANTITY='MASS FRACTION',SPEC_ID='N-HEPTANE'/

**Comment by drjfloyd** [comment, marked as answer]
SPREAD_RATE determines when a specific grid cell starts emitting fuel. HRRPUA+RAMP_Q defines the rate the grid cell emitts fuel where the time used for the RAMP is the time that grid cell started to emit fuel.

**Reply by abc-k** [reply]
Thank you very much.
I understand about RAMP_Q time and SPREAD_RATE.
